### PR TITLE
Review README contents

### PR DESCRIPTION
**Pull request by Victoremepunto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,13 +120,20 @@ Python Snake
...
+The following example shows an _Entity_ class that supports the _delete_ operation for two different contexts, the `UI`
+(front-end) and the `REST` (back-end) contexts. As you can infer, each context requires very different implementations
+to get the entity removed.
+
 
 <table>
 <tr>
-<th> Normal Approach </th>
-<th> varmeth Approach </th>
+<th> ManageIQ </th>
```
Comment (digitronik) [inline README.md:135]: Its not ManageIQ approach. I added this table to show diff between General and varmeth. I think old header names will helpful.

File: README.md [hunk 2]
```diff
@@ -164,8 +195,10 @@ def test_delete_entity(context):
 
 ### Contribute
 
-- Install in the development mode `pip install -e .`
-- Test your changes
+Feel free to create Issues if you find bugs, or go ahead and submit your own Pull Requests.
+
+**Please note**: When submitting new PRs:
+
+- Make sure your code passes all the pre-commit checks:
```
Comment (digitronik) [inline README.md:202]: I would say only `passes all checks` as `nox` will  run `pre-commit` and `tests`

File: README.md (hunk 1 above)
Comment (Victoremepunto) [reply README.md]: Sorry, I just got confused by the previous statement referencing the `ManageIQ` project.

May I then reword this paragraph bit ? I've moved the `ManageIQ` statement to the first README.md lines, being this a project that relates to another one I feel like it's appropriate to mention it there. if you don't want it on the "frontpage" , I'd suggest at least to add it on a separate section, like an "About" or something similar, and leave the reference to the "relative project" there.


**Optional:**
for the example, I know get you want to illustrate how a "Vanilla Python" case compares to `Varmeth` use case. 

I would then suggest something more explicit like  `Vanilla Python` - or `Plain Python` apply , better than `General` ? just to emphasize "this is what you get" against plain python.

Take a look and see if this works better, feel free to suggest more changes if you don't like it !

File: README.md (hunk 2 above)
Comment (Victoremepunto) [reply README.md]: Agree! fixed

File: README.md (hunk 1 above)
Comment (digitronik) [reply README.md]: I like `Plain Python` :+1:

**Pull request merged**